resources/Languages/es_ES.yml

```diff
@@ -0,0 +1,134 @@
+# Aquí puedes seleccionar los mensajes de theSpawn
+
+# es_ES por @MrBlasyMSK
+
+
+#prefijo
+prefix: "§f[§7the§eSpawn§f] §8»§r "
+
+#mensajes de instalación y teletransporte
+#usos: {world}
+spawn-tp: "§aTe has teletransportado con éxito al spawn de este mundo! §7[§b{world}§7]"
+hub-tp: "§aTe has teletransportado con éxito al spawn del hub! §7[§b{world}§7]"
+spawn-set: "§aHas conseguido §7establecer con éxito §ael spawn para este mundo! §7[§b{world}§7]"
+spawn-changed: "§aHas cambiado con éxito §7el spawn de este mundo! §7[§b{world}§7]"
+hub-set: "Has conseguido §7establecer §ael hub en el mundo §b{world}§a!"
+hub-changed: "§aHas conseguido §7cambiar §ael hub al mundo §b{world}§a!"
+
+#mensajes de eliminación
+#usos: none
+spawn-removed: "§aHas eliminado con éxito el spawn de este mundo!"
+hub-removed: "§aHas eliminado con éxito el hub."
+
+#mensajes de alias
+#usos: {alias} {world}
+alias-teleport: "§aFuiste teletransportado al alias §7{alias}§a! §7§b{world}§7]"
+alias-set: "§aHas configurado con éxito el alias §b{alias}§a para el mundo §b{world}§a!"
+alias-command-description: "§r§aAlias para el mundo §b{world}§a!"
+#usos: {alias}
+alias-removed: "§aHas eliminado con éxito el alias §b{alias}§a!"
+
+#mensajes de casas
+#usos: {home}
+home-teleport: "§aHas sido teletransportado con éxito a tu casa §b{home}§a!"
+home-not-exists: "§cTu casa §b{home}§c no existe!"
+home-set: "§aHas guardado con éxito tu hogar §b{home}§a!"
+home-deleted: "§aHas eliminado con éxito tu casa §b{home}§a!"
+home-already-exists: "§cTu casa §b{home}§c ya existe!"
+#usos: {homelist}
+homelist: "§eTus casas: §7{homelist}"
+
```

Comment:
In theSpawn v1.7.0-dev (and newer) there is a new message:
```yaml
#uses: {max-homes}
highest-home-count-reached: "§cYou've already reached the highest count of §r{max-homes}§c homes!"
```
[line 40 and 41](https://github.com/supercrafter333/theSpawn/commit/ef17d314a07d6b10b638fdbe57c90be2da6897cb#diff-0b934b18ed444bc07da12626e22dc539ba047e86c8b8a05c987df1bfb3926a28R40-R41)